daal4py/sklearn/_n_jobs_support.py

```diff
@@ -212,14 +213,16 @@ def init_with_n_jobs(self, *args, n_jobs=None, **kwargs):
             and isinstance(original_class.__doc__, str)
             and "n_jobs : int" not in original_class.__doc__
         ):
-            parameters_doc_tail = "\n    Attributes"
-            n_jobs_doc = """
-    n_jobs : int, default=None
-        The number of jobs to use in parallel for the computation.
-        ``None`` means using all physical cores
-        unless in a :obj:`joblib.parallel_backend` context.
-        ``-1`` means using all logical cores.
-        See :term:`Glossary <n_jobs>` for more details.
+            # Python 3.13 removed extra tab in class doc string
```

Comment:
Curious about this. The docstring is meant to be interpreted by the Sphinx plugin "napoleon", which is a python package. Should thus be independent of python version. Do you have a screenshot of how it ends up looking like if you leave the tab as it is?

Reply:
- `n_jobs` was not added to applicable classes because matching substring of attributes tail was not found
- There is a sklearnex test for n_jobs in docstrings, so it failed initially
- There are many other renderers of docstring like VSCode and other IDEs

Reply:
Thanks. Can confirm that the issue with the docstrings happens with python3.13, and that this PR solves it.